.github/workflows/samples.yml

```diff
@@ -59,32 +59,37 @@ jobs:
       - name: "Testing Samples"
         run: ./build_tools/testing/test_samples.sh
 
-  # TODO(scotttodd): Convert web test scripts to use installed compiler packages and local runtime
-  #                  builds, rather than local compiler builds. The host tools are too slow to build
-  #                  on standard sized runners (and for casual users).
-
-  # web:
-  #   runs-on: ubuntu-20.04
-  #   # TODO: remove dependence on magic matching build dir names
-  #   env:
-  #     HOST_BUILD_DIR: build-host-install
-  #   steps:
-  #     - name: "Checking out repository"
-  #       uses: actions/checkout@v4.1.7
-  #       with:
-  #         submodules: true
-  #     - name: "Building host tools"
-  #       run: |
-  #         ./build_tools/github_actions/docker_run.sh  \
-  #           --env "CCACHE_NAMESPACE=gcr.io/iree-oss/base@sha256:dc314b4fe30fc1315742512891357bffed4d1b62ffcb46258b1e0761c737b446" \
-  #           gcr.io/iree-oss/base@sha256:dc314b4fe30fc1315742512891357bffed4d1b62ffcb46258b1e0761c737b446 \
-  #           ./build_tools/cmake/build_host_tools.sh "${HOST_BUILD_DIR}"
-  #     - name: "Testing Experimental Web Samples"
-  #       env:
-  #         IREE_EMPSCRIPTEN_BUILD_DIR: build-emscripten
-  #       run: |
-  #         ./build_tools/github_actions/docker_run.sh  \
-  #           --env "IREE_HOST_BUILD_DIR=${HOST_BUILD_DIR}" \
-  #           --env "IREE_EMPSCRIPTEN_BUILD_DIR=${IREE_EMPSCRIPTEN_BUILD_DIR}" \
-  #           gcr.io/iree-oss/emscripten@sha256:2dd4c52f1bb499ab365aad0111fe5538b685d88af38636b409b0cf6a576ab214 \
-  #           ./experimental/web/build_and_test_samples.sh
+  web:
+    runs-on: ubuntu-20.04
+    container: gcr.io/iree-oss/emscripten@sha256:2dd4c52f1bb499ab365aad0111fe5538b685d88af38636b409b0cf6a576ab214
```

Comment:
May be able to drop this in favor of https://github.com/mymindstorm/setup-emsdk , then there would be no need for these Docker workarounds
```yml
    defaults:
      run:
        shell: bash
    steps:
      - name: "Mark git safe.directory"
        run: git config --global --add safe.directory '*'
```

Then we could delete https://github.com/iree-org/iree/blob/main/build_tools/docker/dockerfiles/emscripten.Dockerfile too.

Reply:
After a short look at the action, this seems worth a try :)